exporter/stats/stackdriver/stackdriver.go

```diff
@@ -40,6 +40,8 @@ import (
 	"google.golang.org/grpc/codes"
 )
 
+const userAgent = "opencensus-go-v0"
```

Comment:
Can we make this a constant to share between stats/trace? maybe something in opencensus/common(internal)? I am worried that this will get out of sync.

Reply:
Moved it to internal.